dapr/clients/grpc/client.py

```diff
@@ -942,11 +942,12 @@ def get_configuration(
             items=response.items,
             headers=call.initial_metadata())
 
-    async def subscribe_configuration(
```

Comment:
This was async before for a long time (way before I introduced an async client this week) - but watching a configuration synchronously doesn't make a ton of sense.

Making this synchronous but also handling the handler pattern are breaking changes - however that should be ok given the component wasn't stable.